src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1139,6 +1139,25 @@ void fightBombingRaids(final IDelegateBridge delegateBridge, Supplier<Collection
     }
   }
 
+  public void fightAutoKills(final IDelegateBridge delegateBridge) {
+    // Kill undefended transports. Done here to remove potentially dependent sea battles below
+    for( final Territory t : getPendingBattleSites(false) ) {  // Loop through normal combats i.e. not bombing or air raid
```

Comment:
I don't have intellij.

I'm getting a newer computer.  It seems my desktop shat itself and I'm living with what My laptop can do for another week or so. 

I hate that line wrapping though!